common/record.js

```diff
@@ -0,0 +1,35 @@
+(function () {
+    'use strict';
+
+    angular.module('chaise.recordDisplay', [])
+
+    .directive('recordDisplay', ['ErrorService', function(ErrorService) {
+        return {
+            restrict: 'E',
+            transclude: true,
+            scope: {
+                record: '=',
+                table: '='
+            },
+            templateUrl: '../common/templates/record.html',
+            link: function(scope) {
+                scope.$watch('record', function(newVal, oldVal) {
```

Comment:
So I've been thinking about how to circumvent the use of watches; what do you think about wrapping your directive with an `ng-if="$rootScope.record"` (or `ng-show`?) so that the directive can wait for the promise in the `.run` block to resolve `$rootScope.record`?